### DISCUSSION TITLE
Error sending request for url

### DISCUSSION THREAD

**Discussion by FudeezValo** [Cyfrin/foundry-full-course-cu#719] [General]
Foundry-f23 
Yt link: 
https://www.youtube.com/watch?v=sas02qSFZ74&t=9950s

Lesson 8: html-fund-me-f23

Also help me for Windows 11:
Extentsion that we are using html-fund-me-f23 lesson 8

make anvil is running properly

not able to resolve it, had some firewall issue or something if anyone can help!!  

Below is the error
'''
root@Zorojuro:~/foundry-f23/foundry-fund-me-f23# make deploy
Error: 
error sending request for url (http://localhost:8545/): error trying to connect: tcp connect error: Connection refused (os error 111)

Context:
- Error #0: error trying to connect: tcp connect error: Connection refused (os error 111)
- Error #1: tcp connect error: Connection refused (os error 111)
- Error #2: Connection refused (os error 111)
make: *** [Makefile:43: deploy] Error 1
root@Zorojuro:~/foundry-f23/foundry-fund-me-f23# 
'''

**Comment by theirrationalone** [comment]
Please try to restart your vscode or update wsl or try to change port to 31337 or 1337...
Yes it seems like it's a firewall network access issue.

**Reply by FudeezValo** [reply]
Thank you for your suggestion! Mate 
Its sorted
